.github/actions/setup/action.yml

```
@@ -13,7 +13,7 @@ runs:
       with:
         cache: npm
         registry-url: "https://registry.npmjs.org"
-        node-version: "18.20.1"
+        node-version: "22.3.0"
```

```suggestion
        node-version-file: "package.json"
```

https://github.com/actions/setup-node/blob/1e60f620b9541d16bece96c5465dc8ee9832be0b/docs/advanced-usage.md#node-version-file

---

Or perhaps it refers package.json by default

---

> The node-version input is optional. If not supplied, the node version from PATH will be used. However, it is recommended to always specify Node.js version and don't rely on the system one.

It's probably better to specify the version?